### PR TITLE
vim-patch:b7398fe41c9e

### LINKED ISSUE #23515: Wrong file type for netrw
### Problem

After opening a file from netrw, switching back to netrw itself using `:Ex` displays netrw using the opened file type, for example opening a `.yml` file will display `filetype=yaml` using command `:set ft`
Attaching a screenshot with the problem
image
Also found this on reddit https://www.reddit.com/r/neovim/comments/12ye5rc/lsp_errors_in_netrw_windows/ that i think is related

Currently using version
NVIM v0.10.0-dev-278+ga72e795ed
Build type: Release
LuaJIT 2.1.0-beta3

The problem arose a couple weeks ago, i usually build daily from nightly tag

### Steps to reproduce

1. open a folder using nvim .
2. enter any file with an extension listed (for example test.yml)
3. use the command `:Ex` to go back to netrw
4. the file type is no longer set to `netrw` and the buffer is used like a normal file

### Expected behavior

Switching back to netrw should work just fine

### Neovim version (nvim -v)

NVIM v0.10.0-dev-278+ga72e795ed

### Vim (not Nvim) behaves the same?

Not sure, I don't have it installed

### Operating system/version

5.15.108-1-MANJARO

### Terminal name/version

kitty 0.28.1

### $TERM environment variable

xterm-256color

### Installation

build from repo using tag nightly

### PR DESCRIPTION
**Pull request by clason**
Fix #23515

Update runtime files

https://github.com/vim/vim/commit/b7398fe41c9e1e731d058105a34158871ee83e3f

Co-authored-by: Bram Moolenaar <Bram@vim.org>

### CODE REVIEW

File: runtime/doc/testing.txt [hunk 1]
```diff
@@ -137,8 +138,9 @@ assert_false({actual} [, {msg}])			*assert_false()*
 		Also see |assert-return|.
 		A value is false when it is zero. When {actual} is not a
 		number the assert fails.
-		When {msg} is omitted an error in the form
-		"Expected False but got {actual}" is produced.
+		The error is in the form "Expected range {lower} - {upper},
+		but got {actual}".  When {msg} is present it is prefixed to
+		that.
```
Comment (seandewar) [inline runtime/doc/testing.txt:143]: this is for `asserts_inrange`

File: runtime/doc/testing.txt [hunk 2]
```diff
@@ -134,11 +135,11 @@ assert_fails({cmd} [, {error} [, {msg} [, {lnum} [, {context}]]]])
 assert_false({actual} [, {msg}])			*assert_false()*
 		When {actual} is not false an error message is added to
 		|v:errors|, like with |assert_equal()|.
+		The error is in the form "Expected False but got {actual}".
+		When {msg} is present it is prepended to that.
 		Also see |assert-return|.
 		A value is false when it is zero. When {actual} is not a
```
Comment (seandewar) [inline runtime/doc/testing.txt:141]:
```suggestion

		A value is false when it is zero. When {actual} is not a
```

**Comment by seandewar** [comment]
GitHub won't let me select it, but I forgot the updated header comment in xpm.vim.

File: runtime/syntax/xpm.vim [hunk 3]
```diff
@@ -5,6 +5,8 @@
 " Version:	5.4n.1
```
Comment (seandewar) [inline runtime/syntax/xpm.vim:5]: Version and last change date need updating too.
Also, the "omit" line in the commit message can be removed now.

Comment (clason) [reply runtime/syntax/xpm.vim:5]: Will do that when I squash!

**Review by seandewar — APPROVED** [review]
LGTM I think :+1:

**Review by zeertzjq — COMMENTED** [review]
The `script-here` change should be applied to `if_pyth.txt`

**Review by zeertzjq — APPROVED** [review]

**Pull request merged**